src/log.hpp

```diff
@@ -0,0 +1,22 @@
+#ifndef CNT4007_LOG_HPP
+#define CNT4007_LOG_HPP
+
+#include <fstream>
+
+
+class Log {
+    public:
+	ofstream logMsg;

```

Comment:
This should be private. Its probably better to write a non-static class where ofstream is a member attribute, because otherwise we'd have to pass the log file path into every static function.